src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-get-started-intro.mdx

```diff
@@ -17,6 +17,12 @@ redirects:
 
 You have a variety of options to get started with OpenTelemetry and New Relic. You can jump right into the monitoring of your own apps, hosts, or Kubernetes clusters, or you can set up some demo apps to experiment with OpenTelemetry and New Relic. We have range of instructions that'll help you harness the power of OpenTelemetry to gather data and send it to New Relic.
 
+## Check out our guide to getting started [#guide]
+
+First, we recommend checking out our [Best practice guide for setting up OpenTelemetry and New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practice-setup). 
+
+For docs about specific parts of the implementation process, keep reading.
```

Comment:
@zuluecho9 these other docs aren't so much about specific parts of the implementation process. How about this sentence: "Our best practices guide gets you ready to instrument your own apps or services, but you can also see the sections below for tutorials and other types of installations."